src/modules/hip/kernel/subtract_scalar.hpp

```diff
@@ -0,0 +1,114 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+
+
+__global__ void subtract_scalar_ncdhw_hip_tensor(float *srcPtr,
+                                                 uint3 srcStridesCDH,
+                                                 float *dstPtr,
+                                                 uint3 dstStridesCDH,
+                                                 int channels,
+                                                 float subtractParam,
+                                                 RpptROI3DPtr roiGenericPtrSrc)
+{
+    int id_x = (hipBlockIdx_x * hipBlockDim_x + hipThreadIdx_x) * 8;        // W - inner most dim vectorized
```

Comment:
I see that we are following a mix of camel and snake_case for variable names. Going forward, we need to follow Google coding style for newly implemented functions